### DISCUSSION TITLE
Webradio coverart

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#84] [Fixed Issues]
@rern 

The Readme text for RAudio-1 says 'WebRadio coverarts - online fetched' - how does this happen ?  Do I need to perform some sort of update once my webradio urls are  added ?

Chris

**Comment by Roland-B** [comment]
@ceejayemm

I believe this could mean that RAudio-1 is capable of getting the cover via the stream if they provide it as part of the stream. Not many streams have it to be honest. A good example is "Radio Paradise - Main Mix" which is provided as default in the build.

**Comment by rern** [comment]
[Radio Paradise](https://radioparadise.com/) stations provide coverarts themselves.
Others fetch from [last.fm](last.fm) based on each artist and title which sometimes not available.
